docs/can-view-import.md

```diff
@@ -16,6 +16,30 @@ Statically import a module from within a [can-stache] template. *MODULE_NAME* wi
 
 @param {moduleName} [MODULE_NAME] A module that this template depends on.
 
+@signature `<can-import from="MODULE_NAME" module.default:to="VAR_NAME" />`
+
+Statically import a module from within a [can-stache] template. *MODULE_NAME* will be imported before the template renders. *VAR_NAME* will be set during initial template rendering.
+
+```
+<can-import from="app/helpers/properCase" module.properCase:to="properCase" />
+{{properCase(person.name)}}
+```
+
+@param {moduleName} [MODULE_NAME] A module that this template depends on.
+@param {varName} [VAR_NAME] A scope variable name that the loaded module will be set to.
```

Comment:
Should there be three params here?

Reply:
hmm the third param being the name of the property of the module being bound to the parent? eg. `default` or `properCase`?

We could but don't format it like that in the previous documentation for `value.default:to="scope.vars.whatever"`. I was just trying to be consistent with that.

Reply:
Yeah, it's strange with these "signatures" since they're not actually functions. It seems like these might be two different signatures:

> @signature `<can-import from="MODULE_NAME" module.default:to="VAR_NAME" />`

> @signature `<can-import from="MODULE_NAME" module.EXPORT_NAME:to="VAR_NAME" />`

Or maybe just document the second way and mention that for default exports use `module.default:to`.

I'm not totally sure what would make the most sense to users.

Reply:
I'll go ahead with the second suggestion and just describe how this should be used when referencing the default export.